apps/angular/src/app/header/header.component.ts

```diff
@@ -1,9 +1,34 @@
-import { Component } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, DestroyRef } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { AuthService } from '@js-camp/angular/core/services/auth.service';
 
 /** Header component. */
 @Component({
 	selector: 'camp-header',
 	templateUrl: './header.component.html',
 	styleUrls: ['./header.component.css'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class HeaderComponent {}
+export class HeaderComponent {
+	/** User authorization status. */
```

Comment:
"True" or "false" - it is not a user status. Let's improve docs: 
```suggestion
	/** Whether user is authorized or not. */
```